### DISCUSSION TITLE
umi4 如何开启 jsx runtime automatic

### DISCUSSION THREAD

**Discussion by 111xumengze** [umijs/umi#11814] [Q&A]
ReferenceError: React is not defined

**Comment by 111xumengze** [comment]
extraBabelPlugins: [
    [
      '@babel/plugin-transform-react-jsx',
      {
        runtime: 'automatic'
      }
    ]
  ]

**Reply by fz6m** [reply]
默认情况就是 jsx `automatic` ，检查下自己 `tsconfig.json` 有没有覆盖 `jsx` 的选项吧，如还有问题，需要给一个最小复现。
